config/defaults.config.php

```diff
@@ -22,7 +22,7 @@
 
 		// pass (String)
 		// Database user password. Please don't use empty passwords.
-		'password' => '',
+		'password' => new \ParagonIE\HiddenString\HiddenString(''),
```

Comment:
Additionally, does this mean all local configs have to be changed to a `HiddenString`?